### PR TITLE
Add weekly stale page cleanup command

### PR DESCRIPTION
**Pull request by isAdamBailey**
## Summary
- add a weekly `pages:cleanup-stale` console command to remove pages with `read_count = 0` older than 30 days and delete related S3 assets
- delete books that become empty after stale page cleanup
- schedule the command in the console kernel and add feature tests covering page, asset, and empty-book cleanup behavior

## Test plan
- [x] `sail test tests/Feature/Console/CleanupStalePagesTest.php`
- [x] `sail pint app/Console/Commands/CleanupStalePages.php app/Console/Kernel.php tests/Feature/Console/CleanupStalePagesTest.php`
- [x] `sail test`

Made with [Cursor](https://cursor.com)

### CODE REVIEW

File: app/Console/Commands/CleanupStalePages.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        $path = $storedValue;
+        if (Str::startsWith($storedValue, 'https://')) {
+            $parsedPath = parse_url($storedValue, PHP_URL_PATH);
+            if (! is_string($parsedPath) || $parsedPath === '') {
+                return 0;
+            }
+
+            $path = ltrim($parsedPath, '/');
+        }
```
[the remark this answers is not in the document]
Comment (isAdamBailey) [reply app/Console/Commands/CleanupStalePages.php:104]: @copilot apply changes based on [this feedback](https://github.com/isAdamBailey/shudderfly/pull/83#discussion_r3214021953)

File: tests/Feature/Console/CleanupStalePagesTest.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        $stalePage = Page::factory()->for($book)->create([
+            'read_count' => 0.0,
+            'created_at' => now()->subDays(31),
+            'media_path' => 'books/test/stale-image.webp',
+            'media_poster' => 'books/test/stale-poster.webp',
+        ]);
+        $recentUnreadPage = Page::factory()->for($book)->create([
+            'read_count' => 0.0,
+            'created_at' => now()->subDays(29),
+        ]);
```
[the remark this answers is not in the document]
Comment (isAdamBailey) [reply tests/Feature/Console/CleanupStalePagesTest.php:29]: @copilot apply changes based on [this feedback](https://github.com/isAdamBailey/shudderfly/pull/83#discussion_r3214021956)
